docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
   - Click **Continue**.
 - Click the **Save** button and wait for Strapi to restart.
 
-## 5. Manage and add data to Content Type
+## 5. Create another new Content type
+
+Navigate back to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
```

Comment:
Open the [Content Type creation modal](http://localhost:1337/admin/plugins/content-type-builder/#create::contentType::baseSettings) .

Reply:
This is confusing and the only time we refer to any modal. Once Strapi restarts, they need to click either the Content Type Builder menu item or the center column with "+ Add Content Type".

The user is already familiar with the left menu and so I referred to it to avoid confusion.

Reply:
modal is a fairly technical term thus I agree with @davidkartuzinski a new user isn't likely to understand the meaning of that word